ERROR    root:init_helpers.py:90 Failed to capture screenshot: [Errno 2] No such file or directory: 'data/attachments/dev_chrome_130.0.6723.58_error_message_appears_for_first_name.png'
ERROR    root:init_helpers.py:90 Failed to capture screenshot: [Errno 2] No such file or directory: 'data/attachments/dev_chrome_130.0.6723.58_error_message_appears_for_last_name.png'
ERROR    root:init_helpers.py:90 Failed to capture screenshot: [Errno 2] No such file or directory: 'data/attachments/dev_chrome_130.0.6723.58_error_message_appears_for_dob.png'